internal/bgp/frr/config.go

```diff
@@ -264,14 +133,27 @@ func templateConfig(data interface{}) (string, error) {
 				}
 				return false
 			},
-		}).Parse(configTemplate)
+			"dict": func(values ...interface{}) (map[string]interface{}, error) {
+				if len(values)%2 != 0 {
+					return nil, errors.New("invalid dict call")
```

Comment:
nit: wdyt about having the err be a little bit more detailed?  something like "dict expects even amount of values, got %d"?

Reply:
Won't hurt (even though this kind of mistake will be easily caught by CI), will add